### ISSUE TITLE
400 badRequest on google_iam_policy

### ISSUE THREAD

**Issue by devth** [hashicorp/terraform#10650]
Have tried several incarnations of `google_iam_policy` but always get:

```
Error applying IAM policy for project "foo": googleapi: Error 400: Request contains an invalid argument., badRequest
```

when attempting to apply. v0.7.13.

Sample data:

```json
        "google_iam_policy": {
            "ebay_iam": {
                "binding": [
                    {
                        "role": "roles/owner",
                        "members": ["user:devth@devth.com"],
                    }
                ]
            }
        }
```

**Comment by devth** [comment]
This happens when trying to set a user to an owner, which is invalid. See [constraints](https://cloud.google.com/resource-manager/reference/rest/v1/projects/setIamPolicy) when setting IAM policies. Better error messaging, and perhaps a link to this doc from Terraform's docs would be helpful.

**Comment by stack72** [comment]
Closed via #11188
